### DISCUSSION TITLE
Shortcuts opening multiple tabs

### DISCUSSION THREAD

**Discussion by ahopkins** [Ulauncher/Ulauncher#1385] [Q&A]
I have few shortcuts, some with and without arguments. For some strange reason when I launch one it opens two tabs in my browser instead of one. Anyone else experienced this or know what may be going on?

FWIW, I am using `Opera One(version: 109.0.5097.45)`

**Comment by Uzunari** [comment]
Having the same struggle while using
Opera One(version: 110.0.5130.64)
Ulauncher v5.15.7
